apisix/plugins/jwt-auth.lua

```diff
@@ -45,6 +45,7 @@ local consumer_schema = {
     properties = {
         key = {type = "string"},
         secret = {type = "string"},
+        param = {type = "string", default = "jwt"},
```

Comment:
What about using the name "token_name" instead?

Reply:
ok